public/main.js

```diff
@@ -15,11 +17,18 @@ function createWindow() {
     height: 600,
     webPreferences: {
       nodeIntegration: true,
+      webSecurity: false,
     },
   });
 
   mainWindow.loadURL(isDev ? 'http://localhost:8080' : `file://${__dirname}/../dist/index.html`);
 
+
+  if (isDev) {
+    BrowserWindow.addDevToolsExtension(
+      path.join(os.homedir(), './Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.6.0_0'),
```

Comment:
As we discussed, it’d probably be best to move this elsewhere for production, but looks fine for now 👌

Reply:
👍